.github/workflows/nightly-fuzz.yml

```diff
@@ -35,52 +36,11 @@ jobs:
       - name: Bazel Build Fuzz Driver (opt)
         run: |
           "${GITHUB_WORKSPACE}/bin/bazel" build -c opt xls/fuzzer:run_fuzz_multiprocess
-      - name: Bazel Run Fuzz (nosim, opt)
-        # TODO(leary): 2020-08-28 Can remove some of these "lighter weight"
-        # options as we understand why GitHub action jobs are getting killed.
+      - name: Bazel Run Fuzz (opt)
         run: |
-          "${GITHUB_WORKSPACE}/bin/bazel" run -c opt xls/fuzzer:run_fuzz_multiprocess -- --crash_path "${GITHUB_WORKSPACE}/crashers" --sample_count=1024 --codegen=false --simulate=false --emit_loops=False --summary_path "${GITHUB_WORKSPACE}/crashers"
+          "${GITHUB_WORKSPACE}/bin/bazel" run -c opt xls/fuzzer:run_fuzz_multiprocess -- --crash_path "${GITHUB_WORKSPACE}/crashers" --sample_count=2048 --summary_path "${GITHUB_WORKSPACE}/crashers"
       - name: Upload Fuzz Crashers
         uses: actions/upload-artifact@v2
         with:
           name: crashers
           path: "${GITHUB_WORKSPACE}/crashers"
-  cleanup:
```

Comment:
I think we still want to keep the cleanup.  Even with the faster machines, the cache is still useful for even more speedup.

Reply:
This still use the cache, it's just that we don't need to work around the invalidation logic anymore because a cache miss would still make the build complete in a reasonable time.

Reply:
The idea is that Github's default invalidation logic doesn't work for us.  If we don't have this then our cache will be filled with caches created from CL checkins and the nightly builds will likely not have a cache to use.

This logic keeps the caches clean -- we'll only have the latest cache for each OS around (assuming Github doesn't evict us for going over their 10GB limit).

Reply:
> nightly builds will likely not have a cache to use

would it make sense to use a different cache key for the nightly build? That way its eviction lifecycle would be decouple from pending PRs.

Reply:
Additionally, even without caching the nightly build would still complete in a reasonable time without timeoutong (<1 hour) now that it leverages larger runner.

Reply:
Another reason I'm disabling the cache cleanup in this PR, is that #925 highlighted some issue between the current `gh` tool and the larger runner runtime environment, see https://github.com/actions/gh-actions-cache/issues/56#issuecomment-1499829778

Reply:
There should already be two cache keys in use (and the cleanup rule is so that the the less-used cache key isn't evicted by the more often used one).  Github eviction policy is global and not on a per-key basis.

But if this doesn't work on the larger runners, then ok to remove.